Cryptoscam.txt

```diff
@@ -0,0 +1,5 @@
+text in here
+
+no scam, real money easy here
+
+gib moneh plis
```

Comment:
You didnt specify currency! Please add a currency, because my cryptofarm just blew up.

Reply:
DONE!